pyodata/v2/model.py

```diff
@@ -234,6 +234,22 @@ def register_type(typ):
             # pylint: disable=unsupported-assignment-operation
             Types.Types[collection_name] = collection_typ
 
+    @staticmethod
+    def overwrite_type(typ):
+        """Add new  type to the type repository as well as its collection variant"""
```

Comment:
This is probably copypasted docstring, does not connect to the method name.

This should definitely, as a new feature around Types, be covered with new unit tests, that it behave as expected.
- new type
- redefinition of existing type
- ...whatever I missed